packages/taiko-client/prover/prover.go

```diff
@@ -366,6 +387,8 @@ func (p *Prover) eventLoop() {
 			reqProving()
 		case <-forceProvingTicker.C:
 			reqProving()
+		case <-forceAggregatingTicker.C:
+			reqAggregation()
```

Comment:
So we only force aggregate tier 0 proofs? is it by design?

Reply:
0 means aggregate proofs of all tiers here.

Reply:
I see, lets add a comment about this then?

Reply:
OK